tinygrad/device.py

```diff
@@ -30,8 +30,9 @@ def default(self) -> Compiled: return self[self.DEFAULT]
   @functools.cached_property
   def DEFAULT(self) -> str:
     device_from_env: Optional[str] = functools.reduce(lambda val, ele: ele if getenv(ele) == 1 else val, self._devices, None)   # type: ignore
```

Comment:
Filter DISK and NPY here

Reply:
i updated the patch, is that what you suggest?

Reply:
No